srcpkgs/x2goclient/template

```diff
@@ -0,0 +1,16 @@
+# Template file for 'x2goclient'
+pkgname=x2goclient
+version=4.1.2.2
+revision=1
+build_style=gnu-makefile
+make_build_target="build_client build_man"
+make_install_target="install_client install_man"
+hostmakedepends="pkg-config qt-devel-tools"
+makedepends="qt-devel libldap-devel libssh-devel libXpm-devel cups-devel"
```

Comment:
Do use qt5, it should also be supported (at least nix is using it). Qt4 will be dropped some time soon (and is ages EOL)

Reply:
Done